actions.yml

```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
     contents: read
     pull-requests: read
     security-events: write
+ridedott/merge-me-action:
+  permissions:
+    contents: write
+shogo82148/actions-check-permissions:
```

Comment:
OK: Yes, we can just ignore.
https://shogo82148.github.io/blog/2021/03/17/actions-check-permissions/